src/main/kotlin/it/pagopa/wallet/domain/wallets/Client.kt

```diff
@@ -0,0 +1,32 @@
+package it.pagopa.wallet.domain.wallets
+
+import it.pagopa.wallet.documents.wallets.Client
+import java.time.Instant
+
+data class Client(val status: Status, val lastUsage: Instant?) {
+    fun toDocument(): Client = Client(status.name, lastUsage?.toString())
+
+    enum class Status {
+        ENABLED,
+        DISABLED
+    }
+
+    sealed interface Id {
+        val name: String
+
+        companion object {
+            fun fromString(name: String): Id = WellKnown.NAMES[name] ?: Unknown(name)
+        }
+    }
+
+    enum class WellKnown : Id {
+        CHECKOUT,
```

Comment:
For now, to avoid unusual requirements, it's better to avoid CHECKOUT. As of now, the only client of the `wallet` is IO.